reference_data/inputs/H2O.xyz

```diff
@@ -0,0 +1,5 @@
+3
+Lattice="75.0 0.0 0.0 0.0 75.0 0.0 0.0 0.0 75.0" Properties=species:S:1:pos:R:3 #=T 19.28600=T cellangstrom=T Bead:=T positionsangstrom=T CELLabcABC:=T 0=T 238=T 90.00000=T 19.04200=T 118.72000=T Step:=T 3.56990=T pbc="T T T"
```

Comment:
For small molecules like this, it would be immensely helpful to add a comment saying where the structure came from (database? geometry optimization? With what method?). I know I've wasted far too much time looking at files labelled `methane.xyz` and asking "...but _which_ methane?"

Reply:
This is a weightier task, as I don't know for some of these myself. Let's address these in another issue.

Reply:
@max-veit Can we resolve this set of requested changes in lieu of a new issue?

Reply:
Ok, see #226